examples/standalone_evaluation_llm_as_judge.py

```diff
@@ -43,6 +45,37 @@
 # [Assistant's Answer]
 # black
 
+add_to_catalog(
+    SequentialOperator(
+        steps=[
+            ExtractWithRegex(
+                field="prediction",
+                regex=r"\[\[([\d]+\.?[\d]*)\]\]",
+                process_every_value=False,
+            ),
+            ExtractWithRegex(
+                field="references",
+                regex=r"\[\[([\d]+\.?[\d]*)\]\]",
+                process_every_value=True,
+            ),
+            StrToFloat(
+                field="prediction",
+                no_match_value=0.0,
+                divide_by_10=True,
+                process_every_value=False,
+            ),
+            StrToFloat(
+                field="references",
+                no_match_value=0.0,
+                divide_by_10=True,
+                process_every_value=True,
+            ),
+        ]
+    ),
+    "processors.extract_score",
```

Comment:
I just calling it "processors.extract_number_from_double_square_brackets ".
So it's fully self explanatory.

Reply:
So for now - let's just use the longer name.

processors.extract_score_from_double_square_brackets